tests/integration/components/o-s-s/select-test.ts

```diff
@@ -31,7 +31,7 @@ module('Integration | Component | o-s-s/select', function (hooks) {
       );
 
       await click('.upf-input div');
-      await click('.upf-infinite-select .upf-infinite-select__item:last-child');
+      await click(document.querySelector('.upf-infinite-select .upf-infinite-select__item:last-child')!);
```

Comment:
Why using `querySelector` now?

Reply:
previously, it was because the dropdown was being portaled into the body, but that's not needed anymore